src/VolumeDrawable.ts

```diff
@@ -403,6 +425,29 @@ export default class VolumeDrawable {
     }
   }
 
+  enablePicking(enabled: boolean, channelIndex: number): void {
+    // TODO delete the whole pickRendering, or just enable/disable it and keep it around?
+    // the current implementation will delete and recreate the pickRendering object
```

Comment:
the idea here is that this wont be done too often, and generally you will have the pick buffer around for a whole session.  at some point we might have reason to promote picking to a feature in vole-app and may want to rethink how users activate the functionality.